### PR TITLE
Remove Guid in sample with Jwt

### PR DESCRIPTION
**Pull request by BrennanConroy**

### CODE REVIEW

**Review by halter73 — APPROVED** [review]

**Review by GrabYourPitchforks — APPROVED** [review]

File: src/SignalR/samples/JwtSample/Startup.cs [hunk 1]
```diff
@@ -17,9 +18,16 @@ namespace JwtSample
 {
     public class Startup
     {
-        private readonly SymmetricSecurityKey SecurityKey = new SymmetricSecurityKey(Guid.NewGuid().ToByteArray());
+        private readonly SymmetricSecurityKey SecurityKey;
         private readonly JwtSecurityTokenHandler JwtTokenHandler = new JwtSecurityTokenHandler();
 
+        public Startup()
+        {
+            var key = new byte[16];
```
Comment (davidfowl) [inline src/SignalR/samples/JwtSample/Startup.cs:26]: Stackalloc 😄

Comment (davidfowl) [reply src/SignalR/samples/JwtSample/Startup.cs:26]: Oh wait, it's being stored in the key 😄

Comment (GrabYourPitchforks) [reply src/SignalR/samples/JwtSample/Startup.cs:26]: Looking at this code makes me wonder if we need a dirt-simple API like:

```cs
byte[] bytes = RandomNumberGenerator.GetBytes(16);
```

Comment (BrennanConroy) [reply src/SignalR/samples/JwtSample/Startup.cs:26]: I was looking for an API like that 😄

Comment (GrabYourPitchforks) [reply src/SignalR/samples/JwtSample/Startup.cs:26]: API proposal up at https://github.com/dotnet/runtime/issues/42769

**Pull request merged**